embrace-android-delivery/build.gradle.kts

```diff
@@ -11,6 +11,8 @@ android {
 apiValidation.validationDisabled = true
 
 dependencies {
+    implementation(libs.okhttp)
+    implementation(libs.moshi)
```

Comment:
IMO this module doesn't need to know about Moshi specifically, we can hide that implementation detail by moving `ApiRequestV2` to the `embrace-android-payload` module.